util/set/float64_set_test.go

```diff
@@ -37,12 +32,12 @@ func (s *float64SetTestSuite) TestFloat64Set(c *check.C) {
 		set.Insert(vals[i])
 		set.Insert(vals[i])
 	}
-	c.Assert(set.Count(), check.Equals, len(vals))
+	assert.Equal(set.Count(), len(vals))
```

Comment:
```suggestion
	assert.Equal(len(vals), set.Count())
```

Be award of the expected and actual position. Well, it is always a problem for testing framework 🤣

Reply:
ditto to others

Reply:
Done via de3915dab5ab97e366bcf634c488a2e123c0f139